sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -70,6 +73,26 @@ public static CosmosDiagnostics createCosmosDiagnostics(DiagnosticsClientContext
         return new CosmosDiagnostics(diagnosticsClientContext);
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static Set<URI> getRegionContacted(CosmosDiagnostics cosmosDiagnostics) {
```

Comment:
Should be getRegionsContacted?

Also should we introduce this as Beta first?

Reply:
Good catch, will update it.

Reply:
Done